### PR TITLE
wallet-ext: multiple accounts in active connections view

### PR DESCRIPTION
**Pull request by pchrysochoidis**
* useExplorerLink hook 
  * move the creation of the explorer link out of the ExplorerLink component
* components to be used in connected dapps view 
  * added DAppInfoCard a box that shows the dapp icon name and link
  * added DAppPermissionsList that shows a list of permissions
  * updated SummaryCard to fix overflow and borders for footer and add a divider option
  * updated WalletListSelect to add disconnect mode, disabled state and allow filtering visible accounts
* add functionality to disconnect only specific accounts
* support multi accounts in connected apps view 
  * allow disconnecting specific account from a dapp
  * some refactoring for SuiApps to
    * connected apps to use also the data from the WalletDapps list
    * simplify Props
    * fix stylings and use new components like Link and Button
    * mint demo nft toasts and using mutation

Playground/WalletApps listing page styling changes

https://user-images.githubusercontent.com/10210143/220197176-e68aff9e-515b-4633-a82d-275c721cd5db.mov

Multi-accounts in Disconnect app view

https://user-images.githubusercontent.com/10210143/220197372-c982fc9e-ba83-4820-896c-c08e902a203e.mov

https://user-images.githubusercontent.com/10210143/220197417-730f9b3b-2fd8-47c7-a95d-6f5aa45856c4.mov

https://user-images.githubusercontent.com/10210143/220197467-b2a682d2-23ed-42dc-a484-66e9c69ed3b6.mov

closes APPS-284

### CODE REVIEW

**Review by mystie711 — APPROVED** [review]

File: apps/wallet/src/ui/app/components/sui-apps/index.tsx [hunk 1]
```diff
@@ -2,176 +2,102 @@
...
-                <>
-                    <div className={st.desc}>
-                        <div className={st.title}>
-                            Builders in sui ecosystem
-                        </div>
+            <div className={st.desc}>
+                <div className={st.title}>Builders in sui ecosystem</div>
+                {ecosystemApps?.length ? (
+                    <>
                         Apps here are actively curated but do not indicate any
```
Comment (Jibz1) [inline apps/wallet/src/ui/app/components/sui-apps/index.tsx:80]: not a blocker we could use Text component here

File: apps/wallet/src/ui/app/components/sui-apps/index.tsx [hunk 2]
```diff
@@ -2,176 +2,102 @@
...
+                    loading={mintMutation.isLoading}
+                    text="Mint an NFT"
+                />
+                <Button
+                    size="tall"
+                    variant="outline"
+                    href={accountOnExplorerHref!}
+                    text="View account on Sui Explorer"
+                    onClick={() => {
+                        trackEvent('ViewExplorerAccount');
```
Comment (Jibz1) [inline apps/wallet/src/ui/app/components/sui-apps/index.tsx:72]: no need for the bracket before track

File: apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx [hunk 3]
```diff
@@ -101,48 +65,39 @@ function SuiApp({
...
             ) : (
                 <ExternalLink
-                    href={pageLink || link}
+                    href={link}
                     title={name}
                     className={st.ecosystemApp}
                     showIcon={false}
-                    onClick={onClickAppLink}
+                    onClick={() => {
+                        trackEvent('AppOpen', {
```
Comment (Jibz1) [inline apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx:93]: same here

**Review by Jibz1 — APPROVED** [review]
Except for the minor nit and connected app issue LGTM

**Comment by pchrysochoidis** [comment]
@Jibz1 about the Text component, we should revisit the components used here anyway to remove scss modules use new UI components etc. Now I just tried to touch as less as possible in interest of time. Will leave this and the function props to be addressed at that time.

About the issue you mean the one we discussed offline right (not something else)?  - When we connect to a site and we have the wallet in the connected view on another tab, the list of connected apps doesn't update ([jira issue](https://mysten.atlassian.net/browse/APPS-535)) . This is an existing behaviour and this PR didn't break it so I think it should be addressed in another PR.

**Pull request merged**